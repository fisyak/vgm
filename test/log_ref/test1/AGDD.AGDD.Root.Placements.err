Info in <TGeoManager::TGeoManager>: Geometry VGM Root geometry, VGM Root geometry created
Info in <TGeoManager::SetTopVolume>: Top volume is world. Master volume is world
Info in <TGeoManager::CheckGeometry>: Fixing runtime shapes...
Info in <TGeoManager::CheckGeometry>: ...Nothing to fix
Info in <TGeoManager::CloseGeometry>: Counting nodes...
Info in <TGeoManager::Voxelize>: Voxelizing...
Info in <TGeoManager::CloseGeometry>: Building cache...
Info in <TGeoNavigator::BuildCache>: --- Maximum geometry depth set to 100
Info in <TGeoManager::CloseGeometry>: 241 nodes/ 4 volume UID's in VGM Root geometry
Info in <TGeoManager::CloseGeometry>: ----------------modeler ready----------------
Info in <TGeoManager::Export>: Exporting VGM Root geometry VGM Root geometry as root file. Optimizations streamed.
