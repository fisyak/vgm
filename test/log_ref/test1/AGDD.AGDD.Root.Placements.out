Info in <TGeoManager::TGeoManager>: Geometry VGM Root geometry, VGM Root geometry created
AGDD geometry will be defined via AGDD and exported to Root

*************************************************************
 Geant4 version Name: geant4-09-02    (19-December-2008)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

Visualization Manager instantiating...
TestPlacements finished
Go to Import
VGM info:   Top volume is world
VGM info:   ImportSection: importing AGDD_composition "world_comp"
VGM info:   ImportComposition "world_comp"
VGM info:   ImportPosition of volume "wheel" cardinality 1 parent "world"
VGM info:   ImportComposition "wheel"
VGM info:   ImportPosition of volume "layerA" cardinality 24 parent "world"
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[5000,0,0]->(0,-0,0)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[4829.63,7.92379e-14,-1294.1]->(2.15997e-18,0.261799,1.64066e-17)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[4330.13,1.53076e-13,-2500]->(9.47236e-18,0.523599,3.53513e-17)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[3535.53,2.16482e-13,-3535.53]->(2.53624e-17,0.785398,6.12303e-17)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[2500,2.65135e-13,-4330.13]->(6.12303e-17,1.0472,1.06054e-16)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[1294.1,2.9572e-13,-4829.63]->(1.75345e-16,1.309,2.28515e-16)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[3.06152e-13,3.06152e-13,-5000]->(-6.12303e-17,1.5708,0)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-1294.1,2.9572e-13,-4829.63]->(3.14159,1.309,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-2500,2.65135e-13,-4330.13]->(3.14159,1.0472,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-3535.53,2.16482e-13,-3535.53]->(3.14159,0.785398,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-4330.13,1.53076e-13,-2500]->(3.14159,0.523599,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-4829.63,7.92379e-14,-1294.1]->(3.14159,0.261799,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-5000,3.74915e-29,-6.12303e-13]->(3.14159,1.22461e-16,3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-4829.63,-7.92379e-14,1294.1]->(3.14159,-0.261799,-3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-4330.13,-1.53076e-13,2500]->(3.14159,-0.523599,-3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-3535.53,-2.16482e-13,3535.53]->(3.14159,-0.785398,-3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-2500,-2.65135e-13,4330.13]->(3.14159,-1.0472,-3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-1294.1,-2.9572e-13,4829.63]->(3.14159,-1.309,-3.14159)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[-9.18455e-13,-3.06152e-13,5000]->(-6.12303e-17,-1.5708,0)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[1294.1,-2.9572e-13,4829.63]->(1.75345e-16,-1.309,-2.28515e-16)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[2500,-2.65135e-13,4330.13]->(6.12303e-17,-1.0472,-1.06054e-16)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[3535.53,-2.16482e-13,3535.53]->(2.53624e-17,-0.785398,-6.12303e-17)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[4330.13,-1.53076e-13,2500]->(9.47236e-18,-0.523599,-3.53513e-17)
VGM info:   ImportingPosition with volume "layerA" ("layerA") trans=[4829.63,-7.92379e-14,1294.1]->(2.15997e-18,-0.261799,-1.64066e-17)
VGM info:   ImportSection: importing AGDD_composition "layerA_comp"
VGM info:   ImportComposition "layerA_comp"
VGM info:   ImportPosition of volume "layerB" cardinality 3 parent "layerA"
VGM info:   ImportingPosition with volume "layerB" ("layerB") trans=[0,100,0]->(0,-0,0)
VGM info:   ImportingPosition with volume "layerB" ("layerB") trans=[0,300,0]->(0,-0,0)
VGM info:   ImportingPosition with volume "layerB" ("layerB") trans=[0,500,0]->(0,-0,0)
VGM info:   ImportSection: importing AGDD_composition "layerB_comp"
VGM info:   ImportComposition "layerB_comp"
VGM info:   ImportPosition of volume "layerC" cardinality 1 parent "layerB"
VGM info:   ImportingPosition with volume "layerC" ("layerC") trans=[0,98,0]->(0,-0,0)
VGM info:   ImportPosition of volume "layerC" cardinality 1 parent "layerB"
VGM info:   ImportingPosition with volume "layerC" ("layerC") trans=[0,-98,0]->(0,-0,0)
Import finished
Go to export
VGM info:   Exporting isotope: 
VGM info:   Isotope: "C_12"    Z = 6  N = 12  A = 12 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "C_13"    Z = 6  N = 13  A = 13 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "H_1"    Z = 1  N = 1  A = 1 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "H_2"    Z = 1  N = 2  A = 2 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "U_235"    Z = 92  N = 235  A = 235 g/mole
VGM info:   Exporting isotope: 
VGM info:   Isotope: "U_238"    Z = 92  N = 238  A = 238 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "AL"  "AL"  Z = 13  N = 0  A = 26.98 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "C"  "C"  Z = 6  N = 12  A = 12.0101 g/mole
     0th isotope: Isotope: "C_12"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.9899%
     1th isotope: Isotope: "C_13"    Z = 6  N = 13  A = 13 g/mole  abundance: 1.0101%
VGM info:   Exporting element: 
VGM info:   Element: "H"  "H"  Z = 1  N = 1  A = 1 g/mole
     0th isotope: Isotope: "H_1"    Z = 1  N = 1  A = 1 g/mole  abundance: 100%
     1th isotope: Isotope: "H_2"    Z = 1  N = 2  A = 2 g/mole  abundance: 0%
VGM info:   Exporting element: 
VGM info:   Element: "N"  "N"  Z = 7  N = 0  A = 14.01 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "O"  "O"  Z = 8  N = 0  A = 16 g/mole
VGM info:   Exporting element: 
VGM info:   Element: "U"  "U"  Z = 92  N = 235  A = 235.3 g/mole
     0th isotope: Isotope: "U_235"    Z = 92  N = 235  A = 235 g/mole  abundance: 90%
     1th isotope: Isotope: "U_238"    Z = 92  N = 238  A = 238 g/mole  abundance: 10%
VGM info:   Exporting element: 
VGM info:   Element: "Vacuum_e"  "Vacuum_e"  Z = 1  N = 0  A = 1.01 g/mole
VGM info:   Exporting material: 
VGM info:   Material: "Air"  density 0.0013 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "N"  "N"  Z = 7  N = 0  A = 14.01 g/mole  Elm mass fraction: 69.9967 %
   1th element: Element: "O"  "O"  Z = 8  N = 0  A = 16 g/mole  Elm mass fraction: 30.0033 %
VGM info:   Exporting material: 
VGM info:   Material: "Basic"  density 2.7 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "AL"  "AL"  Z = 13  N = 0  A = 26.98 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Scintillator"  density 1.032 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "C"  "C"  Z = 6  N = 12  A = 12.0101 g/mole
     0th isotope: Isotope: "C_12"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.9899%
     1th isotope: Isotope: "C_13"    Z = 6  N = 13  A = 13 g/mole  abundance: 1.0101%  Elm mass fraction: 91.5324 %
   1th element: Element: "H"  "H"  Z = 1  N = 1  A = 1 g/mole
     0th isotope: Isotope: "H_1"    Z = 1  N = 1  A = 1 g/mole  abundance: 100%
     1th isotope: Isotope: "H_2"    Z = 1  N = 2  A = 2 g/mole  abundance: 0%  Elm mass fraction: 8.46756 %
VGM info:   Exporting material: 
VGM info:   Material: "Uranium"  density 13.61 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "U"  "U"  Z = 92  N = 235  A = 235.3 g/mole
     0th isotope: Isotope: "U_235"    Z = 92  N = 235  A = 235 g/mole  abundance: 90%
     1th isotope: Isotope: "U_238"    Z = 92  N = 238  A = 238 g/mole  abundance: 10%  Elm mass fraction: 100 %
VGM info:   Exporting material: 
VGM info:   Material: "Vacuum"  density 0 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "Vacuum_e"  "Vacuum_e"  Z = 1  N = 0  A = 1.01 g/mole  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Air"  density 0.0013 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "N"  "N"  Z = 7  N = 0  A = 14.01 g/mole  Elm mass fraction: 69.9967 %
   1th element: Element: "O"  "O"  Z = 8  N = 0  A = 16 g/mole  Elm mass fraction: 30.0033 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Basic"  density 2.7 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "AL"  "AL"  Z = 13  N = 0  A = 26.98 g/mole  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Scintillator"  density 1.032 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "C"  "C"  Z = 6  N = 12  A = 12.0101 g/mole
     0th isotope: Isotope: "C_12"    Z = 6  N = 12  A = 12 g/mole  abundance: 98.9899%
     1th isotope: Isotope: "C_13"    Z = 6  N = 13  A = 13 g/mole  abundance: 1.0101%  Elm mass fraction: 91.5324 %
   1th element: Element: "H"  "H"  Z = 1  N = 1  A = 1 g/mole
     0th isotope: Isotope: "H_1"    Z = 1  N = 1  A = 1 g/mole  abundance: 100%
     1th isotope: Isotope: "H_2"    Z = 1  N = 2  A = 2 g/mole  abundance: 0%  Elm mass fraction: 8.46756 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Uranium"  density 13.61 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "U"  "U"  Z = 92  N = 235  A = 235.3 g/mole
     0th isotope: Isotope: "U_235"    Z = 92  N = 235  A = 235 g/mole  abundance: 90%
     1th isotope: Isotope: "U_238"    Z = 92  N = 238  A = 238 g/mole  abundance: 10%  Elm mass fraction: 100 %
VGM info:   Generation medium for material: 
VGM info:   Material: "Vacuum"  density 0 g/cm3  radlen 0 cm  intlen 0 cm  state "Undefined"  temperature 0 kelvin  pressure 0 atm
   0th element: Element: "Vacuum_e"  "Vacuum_e"  Z = 1  N = 0  A = 1.01 g/mole  Elm mass fraction: 100 %
VGM info:   Exporting volume store: 
VGM info:   Exporting solid: 
VGM info:      Box  "world"  hx = 6000mm  hy = 1000mm  hz = 6000mm
VGM info:      0th volume: world   material: Vacuum
VGM info:   Exporting solid: 
VGM info:      Box  "layerA"  hx = 200mm  hy = 600mm  hz = 500mm
VGM info:      1th volume: layerA   material: Air
VGM info:   Exporting solid: 
VGM info:      Box  "layerB"  hx = 200mm  hy = 100mm  hz = 500mm
VGM info:      2th volume: layerB   material: Uranium
VGM info:   Exporting solid: 
VGM info:      Box  "layerC"  hx = 200mm  hy = 2mm  hz = 500mm
VGM info:      3th volume: layerC   material: Scintillator
VGM info:   ExportPlacements for 0th volume world
VGM info:      0th daughter vol = layerA  simple placement: layerA
VGM info:      1th daughter vol = layerA  simple placement: layerA
VGM info:      2th daughter vol = layerA  simple placement: layerA
VGM info:      3th daughter vol = layerA  simple placement: layerA
VGM info:      4th daughter vol = layerA  simple placement: layerA
VGM info:      5th daughter vol = layerA  simple placement: layerA
VGM info:      6th daughter vol = layerA  simple placement: layerA
VGM info:      7th daughter vol = layerA  simple placement: layerA
VGM info:      8th daughter vol = layerA  simple placement: layerA
VGM info:      9th daughter vol = layerA  simple placement: layerA
VGM info:      10th daughter vol = layerA  simple placement: layerA
VGM info:      11th daughter vol = layerA  simple placement: layerA
VGM info:      12th daughter vol = layerA  simple placement: layerA
VGM info:      13th daughter vol = layerA  simple placement: layerA
VGM info:      14th daughter vol = layerA  simple placement: layerA
VGM info:      15th daughter vol = layerA  simple placement: layerA
VGM info:      16th daughter vol = layerA  simple placement: layerA
VGM info:      17th daughter vol = layerA  simple placement: layerA
VGM info:      18th daughter vol = layerA  simple placement: layerA
VGM info:      19th daughter vol = layerA  simple placement: layerA
VGM info:      20th daughter vol = layerA  simple placement: layerA
VGM info:      21th daughter vol = layerA  simple placement: layerA
VGM info:      22th daughter vol = layerA  simple placement: layerA
VGM info:      23th daughter vol = layerA  simple placement: layerA
VGM info:   ExportPlacements for 1th volume layerA
VGM info:      0th daughter vol = layerB  simple placement: layerB
VGM info:      1th daughter vol = layerB  simple placement: layerB
VGM info:      2th daughter vol = layerB  simple placement: layerB
VGM info:   ExportPlacements for 2th volume layerB
VGM info:      0th daughter vol = layerC  simple placement: layerC
VGM info:      1th daughter vol = layerC  simple placement: layerC
VGM info:   ExportPlacements for 3th volume layerC
Info in <TGeoManager::SetTopVolume>: Top volume is world. Master volume is world
Export finished
Go to save Root geometry
Info in <TGeoManager::CheckGeometry>: Fixing runtime shapes...
Info in <TGeoManager::CheckGeometry>: ...Nothing to fix
Info in <TGeoManager::CloseGeometry>: Counting nodes...
Info in <TGeoManager::Voxelize>: Voxelizing...
Info in <TGeoManager::CloseGeometry>: Building cache...
Info in <TGeoNavigator::BuildCache>: --- Maximum geometry depth set to 100
Info in <TGeoManager::CloseGeometry>: 241 nodes/ 4 volume UID's in VGM Root geometry
Info in <TGeoManager::CloseGeometry>: ----------------modeler ready----------------
Info in <TGeoManager::Export>: Exporting VGM Root geometry VGM Root geometry as root file. Optimizations streamed.
No Geant4 geometry is built.
